data_managers/data_manager_build_kraken_database/tool_dependencies.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <set_environment version="1.0">
+        <environment_variable action="set_to" name="KRAKEN_SCRIPT_PATH">$REPOSITORY_INSTALL_DIR</environment_variable>
```

Comment:
@jmchilton if anyone is not using the TS and installing by it's own, he is probably smart enough to do the `env.sh` dance :)
But yes I'm always in favour of removing old cruft.